src/utils/web-client.js

```diff
@@ -0,0 +1,30 @@
+const Axios = require('axios');
+
+const client = module.exports = Axios.create({
+    baseURL: process.env.API_BASE_URL,
+    responseType: 'json',
+    headers: { common: {} },
+    transformResponse: (data) => {
```

Comment:
I believe this `transformResponse` business can be removed.  It looks like it's catered to some other API than user-pal.

Reply:
1000% copy and pasted this file from another project 🍝

Reply:
🧀